src/rov/flight_controller/src/flight_controller.cpp

```diff
@@ -142,7 +142,6 @@ class FlightController : public rclcpp::Node {
         for(int i = 0; i < NUM_THRUSTERS; i++) {
```

Comment:
As they say in Australia: "Oi Cunt, this 'ere is code duplication along with line 236!"
Might want to refactor if you're feeling frisky